components/EventCard/EventCard.test.js

```diff
@@ -0,0 +1,33 @@
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
+
+import EventCard from ".";
+
+describe("EventCard", () => {
+  it("displays all props", () => {
```

Comment:
I personally would like if you could be a bit more descriptive here :) e.g. renders name, date, time... of an event